JumpscaleLibs/sal/zosv2/billing.py

```diff
@@ -16,17 +16,12 @@ def payout_farmers(self, client, reservation_response):
         :param reservation_response: reservation create response
         :type reservation_response: tfgrid.workloads.reservation.create.1
         """
-        asset = None
-        if client.network == "TEST":
-            asset = ASSET_CODE + ":" + TFT_ISSUER_TEST
-        else:
-            asset = ASSET_CODE + ":" + TFT_ISSUER_PROD
-
         # TODO check the wallet client use the right asset to pay the reservation
 
         transaction_hashes = []
         reservation_id = reservation_response.reservation_id
         escrow_informations = reservation_response.escrow_information
+        asset = reservation_response.escrow_information.asset
```

Comment:
I don't think this is correct. what's the value of `reservation_response.escrow_information.asset` ?
I think it only container the asset code e.g `TFT`. but the `client.transert` method expect something like `TFT:GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3`

Also, are we sure the wallet client raises an error if we try to pay with an asset that it doesn't posses ? We do not want to send money to the void here.

Reply:
no, When I tried with a reservation the asset in the escrow_information returned is in the form `TFT:GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3` like this 
![Screenshot from 2020-04-22 18-09-24](https://user-images.githubusercontent.com/17128393/80099458-c965a500-856e-11ea-8c49-c379a5451034.png)